lms/djangoapps/bulk_user_retirement/tests/test_views.py

```diff
@@ -85,7 +85,7 @@ def test_retirement_for_multiple_users(self):
                 "usernames": f'{self.user3.username},{self.user4.username}'
             })
             assert response.status_code == 200
-            assert response.data == expected_response
+            assert sorted(response.data['successful_user_retirements']) == sorted(expected_response['successful_user_retirements'])  # pylint: disable=line-too-long
```

Comment:
Was already fixed in the master: https://github.com/openedx/edx-platform/commit/d061f64b274c7f12edad5d77cee170e490722b33